lib/pymedphys/__init__.py

```diff
@@ -1,6 +1,6 @@
 """Module docstring."""
 
-from . import dicom, electronfactors, mosaiq, mudensity, trf, wlutz
+from . import dicom, electronfactors, mosaiq, metersetmap, mudensity, trf, wlutz
```

Comment:
mudensity should no longer be imported, shouldn't mudensity get removed?

Reply:
For backwards compatibility. `pymedphys.mudensity.calculate` was part of our API.

See the new `mudensity.py` that this is importing from:

https://github.com/pymedphys/pymedphys/blob/60c4cbda800750022a0d5817c08e30670ce714a1/lib/pymedphys/mudensity.py#L1-L23

Reply:
> If that's the case, then maybe this could have a comment indicating that it is being deprecated and should be removed by version 0.xy ?

Yup, deprecation notice already included. I haven't specified a release in which it'll be removed. I suspect we can have a "last notice" release down the line where all deprecation notices get a "this will be removed in the next version" addition. And then in the following version each deprecation gets removed.